### PR TITLE
Make location item layout multi-line compatible

### LINKED ISSUE #463: Long station names indistinguishable in auto-completion list
I know it's more 9292's fault than Transportrs, but still, this is kinda an issue. Maybe locations could be displayed multiline instead of ellipsed?

### PR DESCRIPTION
**Pull request by TheLastProject**
Fixed #463 

photo_2018-07-18_19-55-39

### CODE REVIEW

**Comment by TheLastProject** [comment]
I accept the agreement.

**Comment by grote** [comment]
@cla-bot[bot] check

File: app/src/main/res/layout/location_item.xml [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
 		android:layout_width="32dp"
 		android:layout_height="32dp"
 		android:padding="3dp"
-		android:scaleType="fitCenter"
 		android:tint="?colorControlNormal"
 		tools:ignore="ContentDescription"
+		app:layout_constraintStart_toStartOf="parent"
+		app:layout_constraintTop_toTopOf="@+id/textView"
+		app:layout_constraintBottom_toBottomOf="@+id/textView"
 		tools:src="@drawable/ic_action_home"/>
```
[the remark this answers is not in the document]
Comment (TheLastProject) [reply app/src/main/res/layout/location_item.xml]: That sadly doesn't seem to make a difference

**Comment by grote** [comment]
I'll merge this now and have a look myself.

File: app/src/main/res/layout/location_item.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (grote) [reply app/src/main/res/layout/location_item.xml]: Problem was that the line of text could be smaller than the icon, so it wasn't centered to the icon and moved up.

**Pull request merged**